pkg/antctl/antctl.go

```diff
@@ -509,6 +509,105 @@ var CommandList = &commandList{
 			},
 			transformedResponse: reflect.TypeOf(recordmetrics.Response{}),
 		},
+		{
+			use:   "clickhouse",
+			short: "Update clickhouse parameters used in flow aggregator",
+			example: `
+  Enable clickhouse
+  $ antctl update clickhouse --enable true/false
+  Update database
+  $ antctl update clickhouse --database name
+  Update databaseURL
+  $ antctl update clickhouse -d Http://xxxxx
+  Update debug
+  $ antctl update clickhouse --debug true
+  Update compress
+  $ antctl update clickhouse --compress true
+  Update commitInterval
+  $ antctl update clickhouse -i 10s
+  Update ttl
+  $ antctl update clickhouse --ttl 100
+`,
+			commandGroup: update,
+			flowAggregatorEndpoint: &endpoint{
+				nonResourceEndpoint: &nonResourceEndpoint{
+					path:       "/clickhouse",
+					outputType: single,
+					params: []flagInfo{
+						{
+							name:            "enable",
+							shorthand:       "",
+							supportedValues: []string{"true", "false"},
+							usage:           "Enable/Disable Clickhouse for exporting flow records",
+						},
+						{
+							name:      "database",
+							shorthand: "b",
+							usage:     "Update the name of database",
+						},
+						{
+							name:      "databaseURL",
+							shorthand: "d",
+							usage:     "Update the URL of the database",
+						},
+						{
+							name:            "debug",
+							shorthand:       "",
+							supportedValues: []string{"true", "false"},
+							usage:           "Enable or disable debug log",
+						},
+						{
+							name:            "compress",
+							shorthand:       "c",
+							supportedValues: []string{"true", "false"},
+							usage:           "Enable or disable lz4 compression",
+						},
+						{
+							name:      "commitInterval",
+							shorthand: "i",
+							usage:     "Set the periodical interval between batch commit of flow records to DB",
+						},
+						{
+							name:      "ttl",
+							shorthand: "",
+							usage:     "Modify table ttl (unit second) for clickhouse",
+						},
+					},
+				},
+			},
+			transformedResponse: reflect.TypeOf(0),
+		},
+		{
+			use:   "flowcollector",
+			short: "Update flow-collector parameters used in flow aggregator",
+			example: `
+  Update/enable externalflowcollectoraddr
+  $ antctl update flowcollector -e <IP>:<port>[:<proto>]
+  Update disable flow-collector
+  $ antctl update flowcollector --disable true
```

Comment:
In order to re-enable the above command should be used? Would it be more intuitive to separate the two - set address with one command, and enable/disable with another?

Reply:
@annakhm Thanks for the suggestion. I've changed it and updated in this PR.